### PR TITLE
fix(runner-ct): correctly scale runner-ct when spec list is hidden

### PR DESCRIPTION
**Pull request by lmiller1990**
When calculating the scale for the AUT `<iframe>`, we should not subtract the width of the spec list if the spec list is closed.

## Before:

image

## After:

image

### CODE REVIEW

**Review by elevatebart — APPROVED** [review]

File: packages/runner-ct/src/app/RunnerCt.tsx [hunk 1]
```diff
@@ -122,7 +120,7 @@ const App: React.FC<AppProps> = observer(
           type: 'js',
           onClick: () => {
             onNavItemClick(0)
-            setIsSpecsListOpen(!isSpecsListOpen)
+            props.state.setIsSpecsListOpen(!props.state.isSpecsListOpen)
```
Comment (elevatebart) [inline packages/runner-ct/src/app/RunnerCt.tsx:123]:
```suggestion
            state.setIsSpecsListOpen(!props.state.isSpecsListOpen)
```

File: packages/runner-ct/src/app/RunnerCt.tsx [hunk 2]
```diff
@@ -140,12 +138,12 @@ const App: React.FC<AppProps> = observer(
 
     function toggleSpecsList () {
       setActiveIndex((val) => val === 0 ? undefined : 0)
-      setIsSpecsListOpen((val) => !val)
+      props.state.setIsSpecsListOpen(!props.state.isSpecsListOpen)
```
Comment (elevatebart) [inline packages/runner-ct/src/app/RunnerCt.tsx:141]:
```suggestion
      state.setIsSpecsListOpen(!props.state.isSpecsListOpen)
```

File: packages/runner-ct/src/app/RunnerCt.tsx [hunk 3]
```diff
@@ -140,12 +138,12 @@ const App: React.FC<AppProps> = observer(
...
     }
 
     function focusSpecsList () {
       setActiveIndex(0)
-      setIsSpecsListOpen(true)
+      props.state.setIsSpecsListOpen(true)
```
Comment (elevatebart) [inline packages/runner-ct/src/app/RunnerCt.tsx:146]:
```suggestion
      state.setIsSpecsListOpen(true)
```

**Comment by lmiller1990** [comment]
@elevatebart is `state.setIsSpecListOpen` really more preferable to `props.state.setIsSpecListOpen`? I think having `props.` makes it much clearer where the method and associated state lives - seeing `state` looks like local state, at least to me.

**Review by elevatebart — DISMISSED** [review]
LGTM

**Review by elevatebart — DISMISSED** [review]
LGTM

**Review by JessicaSachs — APPROVED** [review]

**Pull request merged**